tests/zfs-tests/tests/functional/cli_root/zpool_create/zpool_create_016_pos.ksh

```diff
@@ -41,7 +41,7 @@
 # STRATEGY:
 # 1. delete all devices in the swap
 # 2. create a zpool
-# 3. Verify the creation is successed.
+# 3. Verify the creation was successful
```

Comment:
This needs a period at the end.

Reply:
Done!

Reply:
Done.

Reply:
The others are still inconsistent, but meh.

Reply:
> The others are still inconsistent, but meh.

Uploaded version with full stop removd